Dockerfile

```diff
@@ -1,16 +1,37 @@
 # Build environment
-FROM alpine AS build
```

Comment:
It looks like you've modified the upstream's Dockerfile rather than creating your own. I would recommend saving this file as something like Dockerfile.mayhem, and updating your Docker workflow with "file: Dockerfile.mayhem" (see other comments).